pyaerocom/web/aerocom_evaluation.py

```diff
@@ -1069,6 +1090,7 @@ def _info_from_map_file(filename):
 
     def get_web_overview_table(self):
         """Computes overview table based on existing map files"""
+        iface_names = self.iface_names
```

Comment:
If I see it correctly this will fail if it is called without `run_evaluation` being called before. How about a property decorator method for iface_names:

```python
@property
def iface_names(self):
   return self._check_and_get_iface_names()
```

Reply:
I have actually never used this construct before. Very useful.